🐞 - AbstractTuiControl is not defined

### Reproduction url

https://stackblitz.com/~/github.com/taiga-family/taiga-playground

### Description

Using TuiTable returns a 
✘ [ERROR] Could not resolve "@taiga-ui/legacy/classes"


### Taiga UI version

4.9.0

### Browser Used

Firefox

### OS Used

macOS

--- a/projects/addon-table/components/table/td/td.component.ts
+++ b/projects/addon-table/components/table/td/td.component.ts
@@ -1,6 +1,5 @@
 import {ChangeDetectionStrategy, Component, ContentChild} from '@angular/core';
 import {TuiControl} from '@taiga-ui/cdk/classes';
-import {AbstractTuiControl} from '@taiga-ui/legacy/classes';
 
 @Component({
     standalone: true,
@@ -9,13 +8,10 @@ import {AbstractTuiControl} from '@taiga-ui/legacy/classes';
     styleUrls: ['./td.style.less'],
     changeDetection: ChangeDetectionStrategy.OnPush,
     host: {
-        '[class._editable]': 'current || legacy',
+        '[class._editable]': 'control',
     },
 })
 export class TuiTableTd {
     @ContentChild(TuiControl)
-    protected readonly current: unknown;
-
-    @ContentChild(AbstractTuiControl)
-    protected readonly legacy: unknown;
+    protected readonly control: unknown;
 }

--- a/projects/legacy/classes/control.ts
+++ b/projects/legacy/classes/control.ts
@@ -4,7 +4,7 @@ import {ChangeDetectorRef, DestroyRef, Directive, inject, Input} from '@angular/
 import {takeUntilDestroyed} from '@angular/core/rxjs-interop';
 import type {AbstractControl, ControlValueAccessor} from '@angular/forms';
 import {NgControl, NgModel} from '@angular/forms';
-import {TuiValueTransformer} from '@taiga-ui/cdk/classes';
+import {TuiControl, TuiValueTransformer} from '@taiga-ui/cdk/classes';
 import {EMPTY_FUNCTION} from '@taiga-ui/cdk/constants';
 import {tuiIsPresent, tuiProvide} from '@taiga-ui/cdk/utils/miscellaneous';
 import {
@@ -255,6 +255,6 @@ export abstract class AbstractTuiControl<T>
     }
 }
 
-export function tuiAsControl<T>(control: Type<AbstractTuiControl<T>>): Provider {
-    return tuiProvide(AbstractTuiControl, control);
+export function tuiAsControl<T>(control: Type<AbstractTuiControl<T>>): Provider[] {
+    return [tuiProvide(AbstractTuiControl, control), tuiProvide(TuiControl, control)];
 }

